disko-install

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+showUsage() {
+  cat <<EOF
+Usage: $0 [OPTIONS]
+  -f, --flake FLAKE_URI#ATTR  Use the specified flake to install the NixOS configuration.
+  --dry-run                   Print the commands that would be run, but do not run them.
+  --show-trace                Show the stack trace on error.
+  -h, --help                  Show this help message.
+EOF
+}
+
+serialiaseArrayToNix() {
+  local -n array=$1
+  nixExpr="{ "
```

Comment:
Why not json?

Reply:
Is this not the same complexity regarding escaping required on the serialisation part?